packages/lesswrong/server/vendor/react-html-email/__tests__/StyleValidator.test.js

```diff
@@ -0,0 +1,76 @@
+import StyleValidator from '../src/StyleValidator'
+
+describe('StyleValidator', () => {
+  beforeEach(() => {
```

Comment:
Hmm, we don't have jest. Will this break? Should we remove these tests, given that we can't run them?

Reply:
I don't think we can run this library's unit tests. Our test runner doesn't seem to try, which is good. Might as well leave them here for reference, though.